### DISCUSSION TITLE
custom console log for multiple compiler

### DISCUSSION THREAD

**Discussion by Vanillabacke** [webpack/webpack#15146] [Q&A] [answered]
Hi,
is there a way to create a combined custom console log for both of the configs?

For example:
I'm in watch mode and the second file is saved / compiled, webpack gives me information about that task in the console.
The problem is, that the log of the first config is not attached to the second log.

I would like to clear the console and it should show me the log of both `firstConfig` and `secondConfig`. Further more I would like to get out some of the data of both and create some stats out of that. Like the first file hast compiled with 3.18 kib and the second with 4.8 kib and the log should show for example the sum of both.

```js
const firstConfig = {
    entry: './first/entry',
    output: { /* output */},
    plugins: [/* plugins */]
}
const firstConfig = {
    entry: './second/entry',
    output: {/* different output */},
    plugins: [/* different plugins */]
}

module.exports = [
   firstConfig,
   secondConfig,
]
```

**Comment by alexander-akait** [comment, marked as answer]
You can create custom plugin for this case, we have `done` hook

**Reply by Vanillabacke** [reply]
thanks a lot for that hint!
